### PR TITLE
Update to Kibana v5

### PR DESCRIPTION
**Pull request by zahodi**
- update kibana apt repo
- update default version to 5.0.0
- remove sense plugin install tasks since sense is included by default with kibana.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -17,20 +19,8 @@
 - name: Copy Kibana configuration.
   template:
     src: kibana.yml.j2
-    dest: "/opt/kibana/config/kibana.yml"
+    dest: "/etc/kibana/kibana.yml"
     owner: root
     group: root
     mode: 0644
   notify: restart kibana
-
```
Comment (senorsmile) [inline tasks/main.yml:25]: do we not want the ability to install kibana plugins at all from ansible now?

Comment (zahodi) [reply tasks/main.yml:25]: Maybe in the future. In which case we would rework the install plugin option.

Comment (senorsmile) [reply tasks/main.yml:25]: Should we leave the code in simply commented out for now then?

Comment (zahodi) [reply tasks/main.yml:25]: No, kibana install location and command changed.

Comment (senorsmile) [reply tasks/main.yml:25]: ok

**Review by senorsmile — APPROVED** [review]
:+1:

**Pull request merged**